gcpkms/keysource.go

```diff
@@ -216,6 +238,8 @@ func (key *MasterKey) newKMSClient() (*kms.KeyManagementClient, error) {
 
 	var opts []option.ClientOption
 	switch {
+	case key.tokenSource != nil:
+		opts = append(opts, option.WithTokenSource(key.tokenSource))
```

Comment:
Can you update the comment for this function to also mention `tokenSource` (and not just `credentialJSON` and `grpcConn` etc.)?

Reply:
Fixed, thanks!